src/controllers/ContinuedIllnessController.ts

```diff
@@ -0,0 +1,77 @@
+import Joi from '@hapi/joi';
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+import { BaseController } from './BaseController';
+import { FormValidator } from './validators/FormValidator';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { IllnessReasonFeatureMiddleware } from 'app/middleware/IllnessReasonFeatureMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { Appeal } from 'app/models/Appeal';
+import { Illness } from 'app/models/Illness';
+import { YesNo } from 'app/models/fields/YesNo';
+import { createSchema } from 'app/models/fields/YesNo.schema';
+import { CONTINUED_ILLNESS_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template = 'continued-illness';
+
+const navigation : Navigation = {
+    previous(): string {
+        return '';
+    },
+    next(): string {
+        return '';
+    },
+    actions: (_: boolean) => {
+        return {
+            continue:'action=continue'
+        };
+    }
+};
+
+@controller(CONTINUED_ILLNESS_PAGE_URI, IllnessReasonFeatureMiddleware, SessionMiddleware, AuthMiddleware)
+export class ContinuedIllnessController extends BaseController<any> {
```

Comment:
That `<any>` should be replaced with the type that corresponds to data structure behind HTML from the template. In other wards it should be the data structure of `request.body`.

Reply:
The compiler doesn't seem to like this, and I can't see any instances of this in the codebase. Perhaps it should be Illness instead?
![Screenshot 2020-07-29 at 10 40 38](https://user-images.githubusercontent.com/43062389/88784431-f2a65a80-d187-11ea-8196-59955a7cd158.png)

Reply:
It should be data type behind HTML form! See below how it is used:

```
prepareSessionModelPriorSave(applicationData.appeal || {}, request.body);
```

Reply:
Implemented as discussed over Slack. Marking as resolved.